packages/alert-message/alert-message.scss

```diff
@@ -144,99 +88,19 @@ $success-border-color--inverted: $suksess--darkbg;
     }
 
     &--info {
-        border-color: $info-border-color;
-        border-color: var(--info-border-color);
-        background-color: $info-background-color;
-        background-color: var(--info-background-color);
-
-        .jkl-alert-message__icon {
-            color: $info;
-            color: var(--info-border-color);
-        }
+        background-color: $bg-color--info;
     }
 
     &--warning {
-        border-color: $warning-border-color;
-        border-color: var(--warning-border-color);
-        background-color: $warning-background-color;
-        background-color: var(--warning-background-color);
-
-        .jkl-alert-message__icon {
-            color: $warning-border-color;
-            color: var(--warning-border-color);
-
-            .jkl-icon-exclamation {
-                color: $border-color;
-                color: var(--border-color);
-            }
-        }
+        background-color: $bg-color--advarsel;
     }
 
     &--error {
-        border-color: $error-border-color;
-        border-color: var(--error-border-color);
-        background-color: $error-background-color;
-        background-color: var(--error-background-color);
-
-        .jkl-alert-message__icon {
-            color: $error-border-color;
-            color: var(--error-border-color);
-        }
+        background-color: $bg-color--feil;
     }
 
     &--success {
-        border-color: $success-border-color;
-        border-color: var(--success-border-color);
-        background-color: $success-background-color;
-        background-color: var(--success-background-color);
-
-        .jkl-alert-message__icon {
-            color: $success-border-color;
-            color: var(--success-border-color);
-        }
-    }
-
-    &--dark {
```

Comment:
Dette føles som et naturlig sted å påpeke at alle blokker av denne typen vil kunne slettes når vi dropper støtte for IE 😎